src/externals/pio2/src/clib/ncparser.pl

```diff
@@ -0,0 +1,362 @@
+#!/usr/bin/perl 
+use strict;
+
+my $netcdf_incdir = $ARGV[0];
+my $pnetcdf_incdir = $ARGV[1];
+
```

Comment:
@jgfouca : I don't remember making any of these files on the ACME side. I suspect that the file was added in ACME to the new location when it was moved around ( ACME-Climate/ACME@d0be438b6b3822d8456c7512d7cf86bc2174339a) but was deleted in CIME (ncparser.pl is no longer available in the pio2 repo and CIME brought in a newer version of pio2)